src/reference/config/telemetry/events.md

```diff
@@ -7,34 +7,89 @@ tag:
   - Events
 ---
 
-# Events
+# Telemetry Events
 
-#### Request Accepted
+Named events happening in Zilla.
 
-A request was accepted by the http server.
- 
-#### Authorization Failed
+:::: note Properties
+
+- [Catalog Errors](#catalog-errors)
+- [Filesystem Errors](#filesystem-errors)
+- [Guard Errors](#guard-errors)
+- [HTTP Access](#http-access)
+- [Kafka Errors](#kafka-errors)
+- [Model Errors](#model-errors)
+- [TCP Errors](#tcp-errors)
+- [TLS Errors](#tls-errors)
+
+::::
+
+## Event Logs
+
+### Catalog Errors
+
+> REMOTE_ACCESS_REJECTED :method :url :status
+
+A remote access error happened in the schema registry catalog.
+
+### Filesystem Errors
+
+> FILE_NOT_FOUND :location
+
+No file was found at the specified location.
+
+### Guard Errors
+
+> AUTHORIZATION_FAILED :identity
+
+A client failed authorization for a Guarded route.
+
+### HTTP Access
+
+> REQUEST_ACCEPTED :identity :scheme :method :authority :path
+
+A successful HTTP request.
+
+### Kafka Errors
+
+> AUTHORIZATION_FAILED :identity
 
 An authorization failure happened in the http, mqtt or the kafka binding.
 
-#### API Version Rejected
+> API_VERSION_REJECTED :apiKey :apiVersion
 
 An API version mismatch occurred in the kafka binding.
 
-#### DNS Failed
+### Model Errors
 
-A DNS resolution failure happened in the tcp binding.
+> VALIDATION_FAILED :error
 
-#### Remote Access Rejected
+A payload did not have the required model schema.
 
-A remote access error happened in the schema registry catalog.
+### TCP Errors
 
-#### TLS Failed
-#### Protocol Rejected
-#### Key Rejected
-#### Peer Not Verified
-#### Handshake Failed
+> DNS_FAILED :address
 
-An error occurred in the tls binding.
+A DNS resolution failure.
 
----
+### TLS Errors
```

Comment:
I don't know if there is a better way to describe these since the exception logic will apply.

@attilakreiner @jfallows Is it a security vulnerability to add the Exception reason to the event log?

Reply:
Yeah, we are trying to avoid documenting this in terms of any internal details. 

For example, these should not be dependent on Java terminology like SSLException. 

Recommend that one is rephrased to a generic TLS failure and perhaps put last. 

We might be able to further parameterize some of these to add a bit more context.

Reply:
> Recommend that one is rephrased to a generic TLS failure and perhaps put last.
Agreed

Reply:
> We might be able to further parameterize some of these to add a bit more context.
Could you, @jfallows or @attilakreiner, please update the descriptions for these since you would know more about what would likely cause these events?